lib/scrabble.rb

```diff
@@ -1,13 +1,31 @@
 class Scrabble
-  def point_values
+  def initialize
```

Comment:
You don't need to have an initialize - you could have just kept the point_values method and used its return value to access the hash!